modulefiles/module_base.wcoss_dell_p3

```diff
@@ -49,7 +52,6 @@ module load upp/10.0.6
 module load wgrib2/2.0.8
 setenv WGRIB2 wgrib2
 
-module use -a /gpfs/dell1/nco/ops/nwprod/modulefiles/
```

Comment:
are we sure we don't need this line?  How will `gempak` module be found? Is it in the default `MODULEPATH`?